src/pages/distroSettings/tabs/HostTab/getFormSchema.tsx

```diff
@@ -157,12 +190,127 @@ export const getFormSchema = ({
         arch: {
```

Comment:
I kind of notice that it gets harder and harder to cross-reference the schema with the uiSchema... Maybe we should try to put them in a consolidated object like `{ schema: <stuff here>, uiSchema: <stuff here> }` (and maybe I should do this in my provider PR too 😅)